classes/Context.php

```diff
@@ -456,17 +457,23 @@ public function getTranslatorFromLocale($locale)
         if ($this instanceof Context) {
             try {
                 $containerFinder = new ContainerFinder($this);
-                $containerFinder->getContainer()->get('prestashop.translation.translator_language_loader')
-                    ->setIsAdminContext($adminContext)
-                    ->loadLanguage($translator, $locale, $withDB, $theme);
-            } catch (ContainerNotFoundException $exception) {
+                $container = $containerFinder->getContainer();
+                $translatorLoader = $container->get('prestashop.translation.translator_language_loader');
+            } catch (ContainerNotFoundException | ServiceNotFoundException $exception) {
```

Comment:
Isn't that php 8 only?

Reply:
My bad, for exception it was introduced in 7.1 😄

Reply:
Yep I had doubts as well when PHPstorm suggested this, but it's 7.1 indeed so we can use it safely Even on 178 I guess